eth_keys/main.py

```diff
@@ -21,62 +14,27 @@
 )
 
 
-def backend_property_proxy(name):
-    @property
-    def property_fn(self):
-        backend_property = getattr(self.backend, name)
-        return backend_property
-    return property_fn
-
-
-class KeyAPI(object):
-    backend = None
-
-    def __init__(self, backend=None):
-        if backend is None:
-            pass
-        elif isinstance(backend, BaseECCBackend):
-            pass
-        elif isinstance(backend, type) and issubclass(backend, BaseECCBackend):
-            backend = backend()
-        elif is_string(backend):
-            backend = get_backend(backend)
-        else:
-            raise ValueError(
-                "Unsupported format for ECC backend.  Must be an instance or "
-                "subclass of `eth_keys.backends.BaseECCBackend` or a string of "
-                "the dot-separated import path for the desired backend class"
-            )
+_PublicKey = PublicKey
+_PrivateKey = PrivateKey
+_Signature = Signature
```

Comment:
Maybe add a comment explaining we need those aliases otherwise mypy gets confused?